package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "leadership-board-game",
+  "version": "1.0.0",
+  "description": "In this activity I set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --env=jsdom",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.8",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  },
+  "dependencies": {
+    "css-loader": "^6.7.3",
+    "html-webpack-plugin": "^5.5.1",
+    "lodash": "^4.17.21",
+    "style-loader": "^3.3.2",
+    "webpack-cli": "^5.1.1",
+    "webpack-dev-server": "^4.15.0"
+  }
```

Comment:
- [ ] kindly consider including the Webpack  package in your devDependacies because it takes all the code from your application and makes it usable in a web browser

Reply:
Good day, please but web pack cli package is in the dev dependency or can you elaborate more?